lib/ansible/plugins/action/win_reboot.py

```diff
@@ -174,18 +171,26 @@ def check_uptime():
 
             # reset the connection to clear the custom connection timeout
             try:
-                self._connection.set_options(direct={"connection_timeout": connection_timeout_orig})
+                self._connection.set_option("connecton_timeout", connection_timeout_orig)
```

Comment:
bad place for a typo: s/connecton_timeout/connection_timeout/

Reply:
Sigh this was me seeing if an invalid option would cause another failure, will change back.